test/cypress/integration/compliance/small-business-lending/filing-instruction-guide.js

```diff
@@ -1,207 +1,221 @@
 import { FilingInstructionGuide } from './filing-instruction-guide-helpers';
+import { skipOn } from '@cypress/skip-test'
 
 const fig = new FilingInstructionGuide();
 
-describe( '1071 Filing Instruction Guide (FIG)', () => {
+skipOn( 'staging' , () => {
+
+  describe( 'testing', () => {
+
+    it('should print cypress environment', () => {
+      console.log(Cypress.env());
+      console.log(Cypress.env('ENVIRONMENT'));
```

Comment:
```suggestion
      cy.log(Cypress.env());
      cy.log(Cypress.env('ENVIRONMENT'));
```

Reply:
Whoops. That was totally not intended for commit.